### PR TITLE
Fix flake8 errors and run flake8 in Travis

### PR DESCRIPTION
**Pull request by PlasmaPower**
To fully work, this will require chaosbot's owner @amoffat to create a Travis account and enable it for this repo. Currently I don't think Travis is required for merging, I'll fix that in a later PR. That means that if I did something wrong with Travis, it won't break the repo (hopefully :P).

### CODE REVIEW

**Comment by xyproto** [comment]
Just for the record: line length of 99 is not forbidden in PEP8.

**Comment by PlasmaPower** [comment]
~@xyproto You mean 79? Yeah I'm not sure if we should go with flake8 or pep8. What do you think?~

Edit: I have no idea what I'm talking about

**Comment by PlasmaPower** [comment]
@chaosbot Yep, thanks ~amoffat~ our new robot overlords.

**Comment by xyproto** [comment]
@PlasmaPower No, meant 99. The line length does not have to be 79 under PEP8.

> Some teams strongly prefer a longer line length. For code maintained exclusively or primarily by a team that can reach agreement on this issue, it is okay to increase the nominal line length from 80 to 100 characters (effectively increasing the maximum length to 99 characters), provided that comments and docstrings are still wrapped at 72 characters.

**Comment by PlasmaPower** [comment]
So do we want a max line length of 79? It's the default for flake8.

**Comment by xyproto** [comment]
With the risk of igniting a debate similar to vim vs emacs, I think 99 is better.

**Comment by PlasmaPower** [comment]
I agree. Switching.

**Comment by PlasmaPower** [comment]
@xyproto vim BTW ;)

**Comment by PlasmaPower** [comment]
Okay, line length is now at max 99. I also reverted the line length fixing changes (except one where the line was 136 chars long).

File: github_api/prs.py [hunk 1]
```diff
@@ -90,6 +95,7 @@ def label_pr(api, urn, pr_num, labels):
     path = "/repos/{urn}/issues/{pr}/labels".format(urn=urn, pr=pr_num)
     data = labels
     resp = api("PUT", path, json=data)
+    return resp
```
Comment (reddraggone9) [inline github_api/prs.py]: Why not just return the response directly?

File: memoize/__init__.py [hunk 2]
```diff
@@ -1 +1 @@
-from .decorator import memoize
+from .decorator import memoize  # noqa: F401
```
Comment (PlasmaPower) [inline memoize/__init__.py]: Is this import necessary? I'm not sure.

File: github_api/users.py [hunk 3]
```diff
@@ -8,3 +8,4 @@ def get_user(api, user):
 def follow_user(api, user):
     follow_path = "/user/following/{user}".format(user=user)
     follow_resp = api("PUT", follow_path)
+    return follow_resp
```
Comment (reddraggone9) [inline github_api/users.py]: Same.

File: chaos.py [hunk 4]
```diff
@@ -20,15 +17,14 @@
 import github_api.comments
 
 # Has a sideeffect of creating private key if one doesn't exist already
-import encryption
-
-from github_api import exceptions as gh_exc
+# Currently imported just for the sideeffect (not currently being used)
+import encryption  # noqa: F401
```
Comment (PlasmaPower) [inline chaos.py:21]: With syntax checking and flake8, do we still need to import this?

**Comment by PlasmaPower** [comment]
:tada: First successful Travis build! https://travis-ci.org/chaosbot/Chaos/builds/236571261

File: chaos.py (hunk 4 above)
Comment (reddraggone9) [reply chaos.py:21]: Need? No. Keep it for the side effect? Meh. I'm fine either way. The server  has the [public key available](http://chaosthebot.com/pubkey.txt) now.

File: github_api/prs.py [hunk 5]
```diff
@@ -71,7 +72,8 @@ def formatted_votes_summary(votes, total, threshold):
     vfor = sum(v for v in votes.values() if v > 0)
     vagainst = abs(sum(v for v in votes.values() if v < 0))
 
-    return "with a vote of {vfor} for and {vagainst} against, with a weighted total of {total:.1f} and a threshold of {threshold:.1f}" \
+    return "with a vote of {vfor} for and {vagainst} against, with a weighted total \
+            of {total:.1f} and a threshold of {threshold:.1f}" \
```
Comment (reddraggone9) [inline github_api/prs.py]: I think I can live with the longer max line length, but I'd prefer to still use parentheses for implicit line continuations as [PEP 8 recommends](https://www.python.org/dev/peps/pep-0008/#maximum-line-length).

File: memoize/__init__.py (hunk 2 above)
Comment (reddraggone9) [reply memoize/__init__.py]: [Yes](https://github.com/chaosbot/Chaos/blob/e5213ed5dfe3361327418f4b2dabe785ae15d9b2/patch.py#L13)

**Review by reddraggone9 — CHANGES_REQUESTED** [review]

**Comment by PlasmaPower** [comment]
Anybody else having reviews duplicating client-side? It's really annoying.

**Comment by PlasmaPower** [comment]
@reddraggone9 Fixed comments.

File: memoize/__init__.py (hunk 2 above)
Comment (reddraggone9) [reply memoize/__init__.py]: You could probably also fix this with an assigment to `__all__`. ~~Haven't tested. Gotta go...~~

It works:
screenshot from 2017-05-26 20-16-39
screenshot from 2017-05-26 20-16-48

**Review by reddraggone9 — APPROVED** [review]

File: memoize/__init__.py (hunk 2 above)
Comment (PlasmaPower) [reply memoize/__init__.py]: Fixed, thanks!

**Pull request merged**